src/core/election.ts

```diff
@@ -113,6 +113,7 @@ export abstract class ElectionCore extends TransactionCore {
           },
           censusOrigin: this.censusOriginFromCensusType(election.census.type),
           metadata: cid,
+          maxCensusSize: election.census.size ?? undefined,
```

Comment:
@marcvelmer This variable is not available for setting by the user currently. Is that correct? Is that the future plan?